Collor/Classes/CollectionSectionDescriptable.swift

```diff
@@ -18,19 +18,26 @@ private struct AssociatedKeys {
 }
 
 public protocol CollectionSectionDescribable : Identifiable {
-    func sectionInset(_ collectionView: UICollectionView) -> UIEdgeInsets
-    func minimumInteritemSpacing(_ collectionView: UICollectionView, layout: UICollectionViewFlowLayout) -> CGFloat
-    func minimumLineSpacing(_ collectionView: UICollectionView, layout: UICollectionViewFlowLayout) -> CGFloat    
+    func sectionInset(_ bounds:CGRect) -> UIEdgeInsets
+    func minimumInteritemSpacing(_ bounds:CGRect, layout: UICollectionViewFlowLayout) -> CGFloat
+    func minimumLineSpacing(_ bounds:CGRect, layout: UICollectionViewFlowLayout) -> CGFloat
+}
+
+public extension CollectionSectionDescribable {
```

Comment:
not here! It's a specific extension. This code must be in your own project, not in the library.